### ISSUE TITLE
Get a "Skip to main content" link in the header bar

### ISSUE THREAD

**Issue by noahheck** [noahheck/office-forge#2]
I want users to know that we care about the accessibility of the application. For users not wanting this functionality visible, we can provide an option in their settings area to hide accessibility targets, but they should be present by default.
